docs/source/improve-workflows/finetuning/index.md

```diff
@@ -30,11 +30,11 @@ The NeMo Agent Toolkit provides a powerful finetuning harness designed for **in-
 
 The finetuning harness is built on three foundational principles:
 
-| Principle | Description |
-|-----------|-------------|
+| Principle | Description                                                                                                                          |
+|-----------|--------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
I don't think the field widths need to line up to be properly rendered in markdown.